### PR TITLE
Addition of 'sign up' flow

### PR DESCRIPTION
**Pull request by Nat-A-M**
Screenshot 2022-10-24 at 11 43 40

### CODE REVIEW

File: app/views/account/index.html [hunk 1]
```diff
@@ -8,50 +8,35 @@
...
+    </li>
+    
+  </ul>
 
   {{ govukButton({
-    text: "Continue"
+    text: "Create a GOV.UK account"
+  }) }}
+
+  <p class="govuk-body">If you already have a GOV.UK account you can <a href="./sign-in" class="govuk-link">sign in</a>.</p>
```
Comment (jesseyuen) [inline app/views/account/index.html:29]: This anchor should go to a page that looks like this:
https://ova-alpha.london.cloudapps.digital/govuk_account_sign_in

File: app/views/account/check-your-emails.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      {{ govukInput({
+        label: {
+        text: "Enter the 6 digit security code"
+        },
+        classes: "govuk-input--width-5",
+        id: "width-6",
+        name: "width-6"
+      }) }}
+
+      <p class="govuk-body"> <a href="/application/sorry" class="govuk-link">Request a new code</a> if:</p>
```
Comment (jesseyuen) [inline app/views/account/check-your-emails.html:37]: This link should go to this file but lets also rename that file to something more logical like `request-code` 

```suggestion
      <p class="govuk-body"> <a href="/account/create-account" class="govuk-link">Request a new code</a> if:</p>
```

File: app/views/account/you-are-signed-in.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+{% set pageHeading = "Do you want to search for a thing?" %}
+{% set backLink = "false" %}
+{% set formAction = "/application/start" %}
+{% set signedOut = "true" %}
+
+{% block formContent %}
+
+  <h1 class="govuk-heading-xl">You are signed in to GOV.UK</h1>
+
+  <p class="govuk-body">We will now ask you a few questions about your work experience and qualifications.</p>
```
Comment (jesseyuen) [inline app/views/account/you-are-signed-in.html:13]: Suggested copy update below! 

We probably also need to consider a scenario here where they have already finished an application and have bene accepted as an expert. In that instance the content might be something like  "Continue to your access you account". 

```suggestion
  <p class="govuk-body">Continue to apply to become to be a subject matter expert.</p>
```

**Review by jesseyuen — CHANGES_REQUESTED** [review]
Ive added some thing to look at bud :)

**Review by Nat-A-M — COMMENTED** [review]
Changes have been made

File: app/views/account/check-your-emails.html [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+      {{ govukInput({
+        label: {
+        text: "Enter the 6 digit security code"
+        },
+        classes: "govuk-input--width-5",
+        id: "width-6",
+        name: "width-6"
+      }) }}
+
+      <p class="govuk-body"> <a href="/application/request-code" class="govuk-link">Request a new code</a> if:</p>
```
Comment (jesseyuen) [inline app/views/account/check-your-emails.html:37]: This link was broken. Correct path here:

```suggestion
      <p class="govuk-body"> <a href="/account/request-code" class="govuk-link">Request a new code</a> if:</p>
```

File: app/views/account/index.html [hunk 5]
```diff
@@ -3,55 +3,40 @@
...
```
Comment (jesseyuen) [inline app/views/account/index.html:29]: Link path update:

```suggestion
  <p class="govuk-body">If you already have a GOV.UK account you can <a href="/account/sign-in" class="govuk-link">sign in</a>.</p>
```

File: app/views/account/sign-in.html [hunk 6]
```diff
@@ -8,27 +8,23 @@
...
 
   {{ govukInput({
-    label: {
-      text: "Add your email address",
+    label: { 
       classes: "govuk-label--m"
     },
     hint: {
-      text: "Do not use a work or university email address you might lose access to."
+      text: "We'll use this to check if you have a GOV.UK account"
```
Comment (jesseyuen) [inline app/views/account/sign-in.html:18]: Copy update:

```suggestion
      text: "Enter your email address to sign in."
```

File: app/views/account/sign-in.html [hunk 7]
```diff
@@ -8,27 +8,23 @@
 
 {% block formContent %}
 
-  <h1 class="govuk-heading-l">Create an account</h1>
+  <h1 class="govuk-heading-l">Enter your email address</h1>
```
Comment (jesseyuen) [inline app/views/account/sign-in.html:11]:
```suggestion
  <h1 class="govuk-heading-l">Sign in with your GOV.UK account</h1>
```

**Comment by jesseyuen** [comment]
Having a look now!

Looking good bud. A couple of things:

Have you seen the “If you already have a GOV.UK account you can sign in.” flow in the prototype we are copying? There is a password field on one of the screens! I hadn’t seen it yet, my apologies. 🤦‍♂️ I think we should add these screens into our flow. 

Start the flow here: https://ova-alpha.london.cloudapps.digital/govuk_create_or_sign_in

Also, don't forget to update the links here:

Screenshot 2022-10-31 at 09 20 07

**Review by jesseyuen — APPROVED** [review]

**Review by jesseyuen — APPROVED** [review]
Looks good!

**Pull request merged**